### PR TITLE
sanitizes comments before put in datastore

### PR DESCRIPTION
**Pull request by apluscs**
made PolicyFactory sanitizer static but not final
it worked, but not sure if this is best practice

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -45,10 +47,12 @@ public Comment(String email, String comment, String date){
   }
   private DatastoreService datastore;
   private static final String COMMENT_DATE_FORMAT = "MMM dd,yyyy HH:mm";
+  private static PolicyFactory sanitizer;
```
Comment (acmcarther) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:50]: You don't want `static` for this unless you're willing to initialize it on this line (and make it final).

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -45,10 +47,12 @@ public Comment(String email, String comment, String date){
   }
   private DatastoreService datastore;
   private static final String COMMENT_DATE_FORMAT = "MMM dd,yyyy HH:mm";
```
Comment (acmcarther) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:49]: Nit: Move this static final above the non-static non-final variables

Comment (apluscs) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:48]: Sure, is are there other rules similar to this one l should know about?

Comment (acmcarther) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:48]: If "similar" means "ordering of class variables", then "sort of". I think there's a specific defined ordering for the combination of [`private|public`, `static`, `final`], probably with public, static, and final at the top (with I think the order being the same as that as well).

If you mean more broadly "are the other formatting rules", then yes, there are:
https://google.github.io/styleguide/javaguide.html

**Pull request merged**